### ISSUE TITLE
loss and mAP values

### ISSUE THREAD

**Issue by ssetty** [experiencor/keras-yolo3#152]
How do we interpret below logs ?  I need to know loss function value, mAP.

loss xy, wh, conf, class:       [22][6.48286247][3.32911205][95.2175751][0.159443796]
avg_obj                 [44][0.826710463]
avg_noobj               [44][0.00814191345]
avg_iou                 [44][0.77933687]
avg_cat                 [44][0.948693633]
recall50        [44][0.820491791]
recall75        [44][0.641009212]
count   [44][39]
